vue-frontend/src/components/home-new/CaseStudy.vue

```diff
@@ -1,15 +1,23 @@
 <template>
   <section class="tw-mt-[100px] tw-font-inter-regular">
-    <h1
-      class="tw-text-black-core/[.03] tw-text-[2.5rem] md:tw-text-[3.75rem] xl:tw-text-[5rem] tw-leading-[1.875rem] md:tw-leading-[4.0625rem] xl:tw-leading-[5rem] tw-text-center"
-    >
-      Our Portfolio
-    </h1>
-    <h1
-      class="tw-container tw-mt-[-20px] md:tw-mt-[-50px] tw-pb-9 md:tw-pb-12 tw-text-[1.875rem] tw-leading-[2.438rem] md:tw-text-[2.65rem] md:tw-leading-[3.218rem] lg:tw-text-[3.438rem] lg:tw-leading-[4rem] tw-text-center tw-text-black-core/[0.87] tw-font-roboto-bold"
+    <div
+      v-if="currentRoutePath == '/portfolio'"
+      class="tw-text-center v2-header-text tw-font-futura-bold tw-mt-8 tw-pb-9 md:tw-pb-20 lg:tw-mt-36"
```

Comment:
Let's not use `v2-header-text`, because the line height is not proper. Line height should be greater then text-size